apps/testingaccessibility/src/components/team/buy-more-seats.tsx

```diff
@@ -1,21 +1,84 @@
 import React from 'react'
+import {z} from 'zod'
+import type {FormattedPrice} from '@skillrecordings/commerce-server/dist/@types'
+import {useQuery} from 'react-query'
+import {useDebounce} from '@skillrecordings/react'
 
-const BuyMoreSeats = () => {
-  const handleOnSubmit = () => {}
-  const calculatedPrice = 0
+const buildFormActionPath = (params: {
+  userId: string
+  quantity: number
+  productId: string
+}) => {
+  const {productId, quantity, userId} = params
+
+  const queryParamString = new URLSearchParams({
+    productId,
+    quantity: String(quantity),
+    userId,
+  }).toString()
+
+  return `/api/skill/checkout/stripe?${queryParamString}`
+}
+
+const buyMoreSeatsSchema = z.object({productId: z.string(), userId: z.string()})
+type BuyMoreSeatsProps = z.infer<typeof buyMoreSeatsSchema>
+
+const BuyMoreSeats = (props: BuyMoreSeatsProps) => {
+  const {productId, userId} = buyMoreSeatsSchema.parse(props)
+
+  const [quantity, setQuantity] = React.useState(5)
+  const debouncedQuantity: number = useDebounce<number>(quantity, 250)
+
+  const {data: formattedPrice, status} = useQuery<FormattedPrice>(
+    ['pricing', debouncedQuantity, productId],
+    () =>
+      fetch('/api/skill/prices', {
+        method: 'POST',
+        body: JSON.stringify({
+          productId,
+          quantity: debouncedQuantity,
+          purchases: [],
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      })
+        .then((res) => res.json())
+        .then((formattedPrice: FormattedPrice) => {
+          return formattedPrice
+        }),
+  )
```

Comment:
This fetches pricing data similar to the Buy page, but the `productId` is restricted to the product ID of their existing bulk purchase.

I also stripped out the `couponId` and `upgradeFromPurchaseId` query params from this path because I don't think they are relevant to an additional bulk purchase.

https://github.com/skillrecordings/products/blob/main/apps/testingaccessibility/src/components/pricing.tsx#L192-L204

![CleanShot 2022-10-25 at 13 45 52@2x](https://user-images.githubusercontent.com/694063/197856516-2f2a6ce5-661b-4497-9046-15d55fef8dce.png)